wallet/lib/services/data_stores/local_data_store.dart

```diff
@@ -182,6 +182,14 @@ abstract class LocalDataSource {
   /// Input: [id] This method will take id of the transaction record to be removed
   /// Output: [bool] status of the process is successful or not
   Future<bool> deleteTransactionFailureRecord(int id);
+
+  /// This method will save that user accepts [Terms of Services] & [Privacy Policy]
+  /// Output: [bool] status of operation is successful or not
+  Future<bool> saveUserAcceptPolicies();
+
+  /// This method will return that user accepts [Terms of Services] & [Privacy Policy] or not
```

Comment:
Why [Terms of Services] and [Privacy Policy]  are in blocks they dont represent any class?